tree/tree/inc/TTree.h

```diff
@@ -216,8 +223,11 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
 
    // TTree status bits
    enum EStatusBits {
-      kForceRead   = BIT(11),
-      kCircular    = BIT(12)
+      kForceRead          = BIT(11),
+      kCircular           = BIT(12),
+      kOnlyFlushAtCluster = BIT(13) // If set, the branch's buffers will grow until an event cluster boundary is hit,
```

Comment:
please use ROOT::Detail::TStatusBitsChecker::Check("TTree") and ROOT::Detail::TStatusBitsChecker::Check("TChain") to make sure there is no bit overlap.